_data/projects/CopperheadOS.yml

```diff
@@ -0,0 +1,12 @@
+name: CopperheadOS
+desc: Hardened Android AOSP fork protecting users communications  
+site: https://github.com/copperhead/
```

Comment:
This could be `https://copperhead.co/` which provides a better overview for users. Or perhaps `https://copperhead.co/android/` which seems to summarize the OS work.